src/main/java/service/CalculateService.java

```diff
@@ -0,0 +1,43 @@
+package service;
+
+import model.Calculator;
+
+public class CalculateService {
+
+    private static final Calculator calculator = new Calculator();
+
+    public int plus(String strX, String strY) {
+        double x = parseDouble(strX);
+        double y = parseDouble(strY);
+        return calculator.plus(x, y);
+    }
+
+    public int minus(String strX, String strY) {
+        double x = parseDouble(strX);
+        double y = parseDouble(strY);
+        return calculator.minus(x, y);
+    }
+
+    public int multiply(String strX, String strY) {
+        double x = parseDouble(strX);
+        double y = parseDouble(strY);
+        return calculator.multiply(x, y);
+    }
+
+    public int divide(String strX, String strY) {
+        double x = parseDouble(strX);
+        double y = parseDouble(strY);
+        return calculator.divide(x, y);
+    }
+
+    private double parseDouble(String input) {
+        if (!isNumber(input)) {
+            throw new IllegalArgumentException("[ERROR] input 값 " + input + "은 숫자가 아닙니다.");
```

Comment:
변수를 사용한 문자열 생성시에는 String.format 을 사용해보는 것도 좋을 것 같아요!! 😁

```suggestion
String.format("[ERROR] input 값 '%s' 은(는) 숫자가 아닙니다.", input);
```